README.md

```diff
@@ -28,7 +28,7 @@ ASP.NET Core API Browser is also very [handy](https://docs.microsoft.com/en-us/d
 | [What's new in ASP.NET Core 2.2](/projects/2-2) | 14 | 2.2 |
 | [What's new in ASP.NET Core 2.1](/projects/2-1) | 6 | 2.1 |
 | [What's new in ASP.NET Core 2.0](/projects/2-0) | 11 | Features introduced in 2.0 but samples run on 2.1 |
-| Foundational ASP.NET Core Samples (scroll down) | 133 | 2.1 |
+| [Foundational ASP.NET Core Samples](#foundation-aspnet-core-21-samples) | 133 | 2.1 |
```

Comment:
Can we please add the version number to the title while at it? So it would become "Foundation ASP.NET Core 2.1 Samples" similar to the title it links to, and other items in the same table.

Thanks.

Reply:
Done.